h3/src/tests.rs

```diff
@@ -1,16 +1,28 @@
+// This is to avoid an import loop:
```

Comment:
Neat! Out of curiosity, any particular reason this can't be h3/src/tests/mod.rs?

Reply:
Sure, I defaulted to the other style but I can change it to match.  Apparently there are clippy lints for this (clippy::self_named_module_files/clippy::mod_module_files).